README.md

```diff
@@ -15,37 +15,54 @@ Follow the instructions below to send logs stored on AWS S3 to Rapid 7.
 ## Deploy the script to AWS Lambda using AWS Console
 1. Create a new Lambda function
 
-2. Choose the Python blueprint for S3 objects
+2. Choose the appropriate Python blueprint for S3 objects
 
    ![Choose Blueprint](doc/step2.png)
 
-3. Configure triggers:
+3. Configure function:
+   * Give your function a name
+   * Assign or create a new role that is allowing GetObject on necessary s3 bucket(s).
+   Note: Creating a new one automatically from this UI adds required permissions automatically`
+   
+4. Configure triggers:
    * Choose the bucket log files are being stored in
-   * Set event type "Object Created (All)"
+   * Set event type "All object create events (All)"
    * Tick "Enable Trigger" checkbox
 
-4. Configure function:
-   * Give your function a name
-   * Set runtime to Python 2.7
+5. Click "Create function"
 
-5. Upload function code:
+6. Upload function code:
    * Create a .ZIP file, containing ```r7insight_lambdas3.py``` and the folder ```certifi```
      * Make sure the files and ```certifi``` folder are in the **root** of the ZIP archive
-   * Choose "Upload a .ZIP file" in "Code entry type" dropdown and upload the archive created in previous step
+   * Go to "Function code" section of lambda setup
+   * Choose "Upload a .zip file" in "Code entry type" dropdown and upload the archive created in previous step
+   * Choose your python runtime version from "Runtime" dropdown
+   * Set handler name to ``lambda_function.lambda_handler``
```

Comment:
i think this need to be r7insight_lambdas3.lambda_handler

Reply:
good catch
fixed